tests/showdown.sh

```
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# Set player 1 address
+PLAYER1="0xC84737526E425D7549eF20998Fa992f88EAC2484"
```

Shellcheck (0.10.0)

[warning] PLAYER1 appears unused. Verify use (or export if used externally).

(SC2034)